helper/clientInitialize.go

```diff
@@ -22,11 +22,6 @@ func getAppknoxAccessToken() string {
 		fmt.Println("Use APPKNOX_ACCESS_TOKEN as env.")
 		os.Exit(1)
 	}
-	_, err := CheckToken()
-	if err != nil {
-		PrintError(err)
-		os.Exit(1)
-	}
```

Comment:
This should be added somewhere, probably after setting `client.BaseURL = baseHost` in `getClient` https://github.com/appknox/appknox-go/blob/1.1.3/helper/clientInitialize.go#L52-L53